exercise_perseus_render/assets/src/vue/index.vue

```diff
@@ -1,13 +1,24 @@
 <template>
 
-  <div></div>
+  <div id="exercise-container">
+    <perseus v-if="item" :item="item"></perseus>
+  </div>
 
 </template>
 
 
 <script>
 
-  module.exports = {};
+  const defaultQuestion = require('./question.json');
```

Comment:
I assume this is for test only right? eventually the `question.json` will be passed as props.